include/istio/control/http/report_data.h

```diff
@@ -47,6 +47,10 @@ class ReportData {
   // Get destination ip/port.
   virtual bool GetDestinationIpPort(std::string* ip, int* port) const = 0;
 
+  // Get rbac permissive mode policy attributes.
+  virtual void GetRBACPermissiveAttributes(std::string* resp_code,
+                                           std::string* policy_id) const = 0;
+
```

Comment:
Do we need to do the same for tcp rbac filter?

Reply:
probably in the future, since tcp rbac filter doesn't have metadata associated, I may prefer to hold it until picture is gets clear.